src/modules/split_chat/index.js

```diff
@@ -10,6 +12,14 @@ class SplitChatModule {
             defaultValue: false,
             description: 'Easily distinguish between messages from different users in chat'
         });
+        watcher.on('chat.message', ($el, msg) => {
+            if (settings.get('splitChat') === false) return;
```

Comment:
odd that it's only available when split chat is on. what's the reasoning behind that?